src/test/java/uk/gov/pay/directdebit/util/ResponseDoesNotContainLinkMatcher.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.pay.directdebit.util;
+
+import org.hamcrest.Description;
+import org.hamcrest.TypeSafeMatcher;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public class ResponseDoesNotContainLinkMatcher extends TypeSafeMatcher<List<Map<String, Object>>> {
+
+    private final String rel;
+
+    private ResponseDoesNotContainLinkMatcher(String rel) {
```

Comment:
I wonder if a `ResponseContainsLinkMatcher` (whose result could be inverted) might be more flexible for the future but given we only need the _not_ case for now, let’s leave it as it is.